tools/LogCollector/source/update_mgmt_health_check.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/python
+
+'''
+   OMS Health Check Script for Update Management Linux VM's
+
+   Authors, Reviewers & Contributors :
+                 Tommy Nguyen Azure Automation SE
+                 Atchut Barli Azure Automation Sr. SE
+                 Shipra Malik Ohri Azure Automation Sr. SE
+                 Shujun Liu Azure Automation Sr. SE
+                 Adrian Doyle Sr. Esc. Eng.
+                 Brian McDermott Sr. Esc. Eng.
+
+   Date        : 2018-07-06
+   Version     : 1.0
+   
+'''
+
+import sys
+import os
+import os.path
+import socket
+import datetime
+
+from os import walk
+
+def main(output_path=None):
+
+    output = []
+
+    output.append(check_network() + "\n")
+    output.append(check_oms_agent_installed() + "\n")
+    output.append(check_oms_agent_running() + "\n")
+    output.append(check_oms_admin() + "\n")
```

Comment:
in 50% of the incidents DSC is not running even though  agent works, so its better to add a check for that.